appium/webdriver/appium_service.py

```diff
@@ -182,7 +180,7 @@ def stop(self) -> bool:
         or has been already stopped.
 
         Returns:
-            bool: `True` if the service was running before being stopped
+            `True` if the service was running before being stopped
```

Comment:
Need `bool` or not?
(L198 has bool, but here has no it)

Reply:
Unnecessary for here since `stop() → bool` already has return type.

https://appium.github.io/python-client-sphinx/webdriver.html#webdriver.appium_service.AppiumService.stop